### PR TITLE
Rewrite to ease first time use.

### PR DESCRIPTION
**Pull request by monty241**
Solely first section.

### CODE REVIEW

File: getting-started/configuration.rst [hunk 1]
```diff
@@ -13,37 +13,41 @@ The configuration file is used to specify what assemblies should be obfuscated,
 where to find the dependencies for the assemblies, and where the obfuscated
 assemblies should be saved.
 
-Table of Settings
------------------
+The configuration file is a XML file with top-level element ``Obfuscator``.
```
Comment (lextm) [inline getting-started/configuration.rst]: It should be "an XML".

File: getting-started/configuration.rst [hunk 2]
```diff
@@ -13,37 +13,41 @@ The configuration file is used to specify what assemblies should be obfuscated,
...
-RenameFields        Whether to rename fields (2.2.0+)
-KeepPublicApi       Whether to keep public API out of obfuscation
-HidePrivateApi      Whether to hide private API via obfuscation
-ReuseNames          Whether to reuse obfuscated names
-UseUnicodeNames     Whether to use Unicode characters as obfuscated names
-UseKoreanNames      Whether to use Korean characters as obfuscated names
-HideStrings         Whether to hide strings
-OptimizeMethods     Whether to optimize methods
-SuppressIldasm      Whether to inform ILdism that assemblies are obfuscated
+InPath              Directory containing the input assembly, such as ``c:\\in``.
```
Comment (lextm) [inline getting-started/configuration.rst]: Should be "input assemblies".

File: getting-started/configuration.rst [hunk 3]
```diff
@@ -13,37 +13,41 @@ The configuration file is used to specify what assemblies should be obfuscated,
...
+OutPath             Directory to contain the obfuscated assembly, such as ``c:\\out`.
```
Comment (lextm) [inline getting-started/configuration.rst]: Should be "output assemblies". Also a ` is missing in the end of the line.

File: getting-started/configuration.rst [hunk 4]
```diff
@@ -13,37 +13,41 @@ The configuration file is used to specify what assemblies should be obfuscated,
...
+XmlMapping          Whether the log file should be of XML format.
+KeyFile             Key file path, such as ``c:\folder\key.pfx``.
+KeyContainer        Key container name.
+RegenerateDebugInfo Whether to generate debug symbols for obfuscated assemblies.
+MarkedOnly          Whether to only obfuscate marked items. All items are obfuscated when set to ``false``.
+RenameProperties    Whether to rename properties.
+RenameEvents        Whether to rename events.
+RenameFields        Whether to rename fields (2.2.0+).
+KeepPublicApi       Whether to exclude public types and type members from obfuscation.
+HidePrivateApi      Whether to exclude private types and type members from obfuscation.
```
Comment (lextm) [inline getting-started/configuration.rst]: Should be "include private".

File: getting-started/configuration.rst [hunk 5]
```diff
@@ -56,33 +60,33 @@ the release as part of the Basic Example:
...
-In addition to being usable like macros, there are a few special variables
-that have additional effects. The variable InPath is used when trying to find
-dependencies (the specified path is searched), and the variable OutPath is
-used as the output path for the obfuscated assemblies and the map. If either
-InPath or OutPath is unspecified, they default to the current path (".").
+A few special variables have functionality besides supporting variable expansion:
+
+- The variable ``InPath`` is used when resolving dependencies by searching the specified path. The default of ``Inpath`` is the current working directory (".").
+- The variable OutPath is used as the output path for the obfuscated assemblies and the logfile specified in the variable ``LogFile``. 
+ The default of ``Inpath`` is the current working directory (".").
```
Comment (lextm) [inline getting-started/configuration.rst]: I don't think the changes of this paragraph cover the original key points. Also "The default of Inpath" appears twice now.

File: getting-started/configuration.rst [hunk 6]
```diff
@@ -102,41 +106,40 @@ keep public items. This can be achieved by setting the following combination,
 .. note:: By using above you don't need to set any obfuscation attribute or
    rule.
 
-This is the default setting since 2.2.0.
+This is the default behavior since version 2.2.0.
 
-Another common case is to strip out everything, which can be achieved by
-setting
+Another common case is to obfuscate all types and type members, which you canachieved using
```
Comment (lextm) [inline getting-started/configuration.rst]: Should be "can achieve".

File: getting-started/configuration.rst [hunk 7]
```diff
@@ -102,41 +106,40 @@ keep public items. This can be achieved by setting the following combination,
...
 
 .. code-block:: xml
 
    <Var name="KeepPublicApi" value="false" />
    <Var name="HidePrivateApi" value="true" />
 
-Of course to keep everything we can use
+Of course to obfuscating nothing you can use
```
Comment (lextm) [inline getting-started/configuration.rst]: Should be "to obfuscate".

**Review by lextm — CHANGES_REQUESTED** [review]
Most changes look good. Please address the issues and then I can merge the pull request.

File: getting-started/configuration.rst (hunk 2 above)
Comment (guidol-ive) [reply getting-started/configuration.rst]: Done.

File: getting-started/configuration.rst (hunk 4 above)
Comment (guidol-ive) [reply getting-started/configuration.rst]: Done.

File: getting-started/configuration.rst (hunk 6 above)
Comment (guidol-ive) [reply getting-started/configuration.rst]: Done.

File: getting-started/configuration.rst (hunk 7 above)
Comment (guidol-ive) [reply getting-started/configuration.rst]: Done.

File: getting-started/configuration.rst (hunk 3 above)
Comment (guidol-ive) [reply getting-started/configuration.rst]: Done.

File: getting-started/configuration.rst (hunk 5 above)
Comment (guidol-ive) [reply getting-started/configuration.rst]: Tried to reformulate. Unsure what you mean by not convering original key points. Can you help me what you mean?

**Review by lextm — APPROVED** [review]

**Pull request merged**